components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3264,7 +3193,7 @@ subroutine cloud_water_conservation(qc,qcnuc,dt,    &
 #endif
 
    sinks   = (qcaut+qcacc+qccol+qcheti+qcshd+qiberg)*dt
-   sources = qc + (qcnuc)*dt
+   sources = qc
```

Comment:
This was atrocious coding as it was originally written and gets worse in this PR. The idea is "are sinks larger than available water"... qc is not a source term so shouldn't be a source. I suggest deleting this line and having the following line be "sinks.gt. qc..."

Reply:
OK